[2022-02-09 08:59:15,293] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2020-09-01T23:00:00+00:00 [queued]>
[2022-02-09 08:59:15,304] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2020-09-01T23:00:00+00:00 [queued]>
[2022-02-09 08:59:15,305] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 08:59:15,305] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 08:59:15,305] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 08:59:15,317] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2020-09-01 23:00:00+00:00
[2022-02-09 08:59:15,325] {standard_task_runner.py:52} INFO - Started process 216 to run task
[2022-02-09 08:59:15,332] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'local_to_gcs_task', 'scheduled__2020-09-01T23:00:00+00:00', '--job-id', '464', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingestee_gcs_dag.py', '--cfg-path', '/tmp/tmpkxg_js6a', '--error-file', '/tmp/tmpa_6yok5i']
[2022-02-09 08:59:15,333] {standard_task_runner.py:77} INFO - Job 464: Subtask local_to_gcs_task
[2022-02-09 08:59:15,420] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2020-09-01T23:00:00+00:00 [running]> on host cc021fb126ed
[2022-02-09 08:59:15,489] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 08:59:15,527] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2020-09-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-09-01T23:00:00+00:00
[2022-02-09 08:59:22,247] {python.py:175} INFO - Done. Returned value was: None
[2022-02-09 08:59:22,262] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=local_to_gcs_task, execution_date=20200901T230000, start_date=20220209T085915, end_date=20220209T085922
[2022-02-09 08:59:22,320] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 08:59:22,356] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-09 09:14:25,851] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2020-09-01T23:00:00+00:00 [queued]>
[2022-02-09 09:14:25,864] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2020-09-01T23:00:00+00:00 [queued]>
[2022-02-09 09:14:25,864] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:14:25,864] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 09:14:25,864] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:14:25,895] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2020-09-01 23:00:00+00:00
[2022-02-09 09:14:25,903] {standard_task_runner.py:52} INFO - Started process 274 to run task
[2022-02-09 09:14:25,910] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'local_to_gcs_task', 'scheduled__2020-09-01T23:00:00+00:00', '--job-id', '553', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingesteee_gcs_dag.py', '--cfg-path', '/tmp/tmpwb6gdlp5', '--error-file', '/tmp/tmptm9nj4iw']
[2022-02-09 09:14:25,911] {standard_task_runner.py:77} INFO - Job 553: Subtask local_to_gcs_task
[2022-02-09 09:14:25,997] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2020-09-01T23:00:00+00:00 [running]> on host 75953ff4f3d5
[2022-02-09 09:14:26,055] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 09:14:26,085] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2020-09-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-09-01T23:00:00+00:00
[2022-02-09 09:14:32,176] {python.py:175} INFO - Done. Returned value was: None
[2022-02-09 09:14:32,189] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=local_to_gcs_task, execution_date=20200901T230000, start_date=20220209T091425, end_date=20220209T091432
[2022-02-09 09:14:32,241] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 09:14:32,284] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
